### PR TITLE
[GraFx Studio] Autogrow

### PR DESCRIPTION
**Pull request by BramVerniest**

### CODE REVIEW

**Comment by BramVerniest** [comment]
In this commit, a rearrange of the menu, to alfabetical is also included

File: docs/GraFx-Studio/guides/text-frame/index.md [hunk 1]
```diff
@@ -1,55 +1,97 @@
...
 ![screenshot](copyfit.png)
 
-Choose the minimum and maximum percentage a font size can vary.
+Set the minimum and maximum percentages for font size adjustments.
+
+> Note: When Copyfitting is enabled, Autogrow is disabled.
+
+#### Auto-grow
+
+Enable "Autogrow" to allow the frame to expand as needed to fit content of varying lengths. For instance, records in a dataset might contain differing character counts.
```
Comment (jeroencosyn) [inline docs/GraFx-Studio/guides/text-frame/index.md:75]: Should be "Auto-grow", with a hyphen. Please check all occurrences on this page.

File: docs/GraFx-Studio/guides/text-frame/index.md [hunk 2]
```diff
@@ -1,55 +1,97 @@
...
+#### Copyfitting
+
+Select the text frame and enable "Copyfitting" to adjust font size to fit the frame by allowing slight reduction or growth.
 
 ![screenshot](copyfit.png)
 
-Choose the minimum and maximum percentage a font size can vary.
+Set the minimum and maximum percentages for font size adjustments.
+
+> Note: When Copyfitting is enabled, Autogrow is disabled.
```
Comment (jeroencosyn) [inline docs/GraFx-Studio/guides/text-frame/index.md:71]: Same comment should be added to auto-grow. When Auto-grow is enabled, Copyfitting is disabled.

File: docs/GraFx-Studio/guides/text-frame/index.md [hunk 3]
```diff
@@ -1,55 +1,97 @@
...
+
+When selecting "Left" or "Right," you can also add the opposite direction for bidirectional growth.
+
+![screenshot](autogrow-left-right.png)
+
+When "Up" is selected, you can also add "Down."
+
+To enable Autogrow, select at least one growth direction, or uncheck "Autogrow" to disable it.
+
+Define the minimum and maximum frame size if needed.
```
Comment (jeroencosyn) [inline docs/GraFx-Studio/guides/text-frame/index.md:89]: Maybe good to add that text will start to overflow if the maximum is reached.

File: docs/GraFx-Studio/guides/text-frame/index.md [hunk 4]
```diff
@@ -1,55 +1,97 @@
...
+
```
Comment (jeroencosyn) [inline docs/GraFx-Studio/guides/text-frame/index.md:90]: I would also add that auto-grow does not work for rotated frames. And if auto-grow is enabled, anchoring for that direction is automatically set to a compatible mode.

File: docs/GraFx-Studio/guides/text-frame/index.md [hunk 5]
```diff
@@ -1,55 +1,97 @@
...
+- Text Overflow
+- Copyfitting
+- Auto-grow
+
+![screenshot](autoresize.png)
+
+#### Text Overflow
+
+With no resizing option selected, text that exceeds the frame will trigger a yellow text-overflow indicator below the frame.
```
Comment (jeroencosyn) [inline docs/GraFx-Studio/guides/text-frame/index.md:55]: We usually write "text overflow", without a hyphen.

File: docs/GraFx-Studio/guides/text-frame/index.md [hunk 6]
```diff
@@ -1,55 +1,97 @@
...
+
+![screenshot](overflow.png)
+
+See also [Output Settings](/GraFx-Studio/guides/output/settings/) to define how a VDP job should handle text overflow.
```
Comment (jeroencosyn) [inline docs/GraFx-Studio/guides/text-frame/index.md:59]: We call it "batch output" on the platform instead of VDP job.

**Review by jeroencosyn — APPROVED** [review]

**Pull request merged**